lib/services/database.go

```diff
@@ -295,6 +391,47 @@ func labelsFromRDSCluster(rdsCluster *rds.DBCluster, meta *types.AWS, endpointTy
 	return labels
 }
 
+// labelsFromRDSProxy creates database labels for the provided RDS proxy.
+func labelsFromRDSProxy(rdsProxy *rds.DBProxy, meta *types.AWS) map[string]string {
+	// rds.DBProxy has no TagList.
+	labels := make(map[string]string)
+	labels[types.OriginLabel] = types.OriginCloud
+	labels[labelVpcID] = aws.StringValue(rdsProxy.VpcId)
+	labels[labelAccountID] = meta.AccountID
+	labels[labelRegion] = meta.Region
+	labels[labelEngine] = strings.ToLower(aws.StringValue(rdsProxy.EngineFamily))
+
+	// TargetRole is read/write for default proxy
+	labels[labelTargetRole] = strings.ToLower(rds.DBProxyEndpointTargetRoleReadWrite)
+	return labels
+}
+
+// parseAWSARNResource extracts resource type and resource ID from ARN
+// resource.
+//
+// https://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html
+func parseAWSARNResource(resource string) (resourceType string, resourceID string) {
```

Comment:
AWS Go SDK provides methods for parsing ARNs. Can we use that instead? We already use it in AWS access for example.

Reply:
`arn.Parse` does not split the resource to resource type vs ID. i think this function is taking the `Resource` parsed from `arn.Parse` to parse further. I couldn't find one from AWS SDK does this. I will try look again. And please let know if you recall one does it. thanks